config.js

```diff
@@ -1,22 +1,20 @@
 System.config({
-  "baseURL": "/rights-inspector",
-  "transpiler": "traceur",
-  "paths": {
-    "*": "*.js",
-    "github:*": "jspm_packages/github/*.js",
-    "npm:*": "jspm_packages/npm/*.js"
-  }
-});
+  //baseURL: '/rights-inspector',
```

Comment:
We need to leave that in for when we deploy. I know it's not compatible with dev locally but I'd rather keep the code broken for dev than for prod. Obv we should find a way to fix this nicely.